app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js

```diff
@@ -9,7 +9,7 @@ module.exports = angular
     require('../securityGroup.write.service.js'),
     require('../../../core/region/regionSelectField.directive.js'),
     require('../../../core/account/account.service.js'),
-    require('../../../core/modal/wizard/modalWizard.service.js'),
+    require('../../../core/modal/wizard/v2modalWizard.service.js'),
```

Comment:
From what I see in the code, you _should_ be able to remove this line altogether...